docs/pages/includes/database-access/cloudsql_service_credentials.mdx

```diff
@@ -0,0 +1,19 @@
+The Teleport Database Service must have credentials of `teleport-db-service`
+GCP service account we created in step 3 in order to be able to generate IAM auth
+tokens.
```

Comment:
```suggestion
The Teleport Database Service must have credentials for the `teleport-db-service`
GCP service account we created in step 3 in order to generate IAM auth
tokens.
```

Reply:
or possibly even just drop the details about why it needs credentials exactly, e.g.:

```suggestion
The Teleport Database Service must have credentials for the `teleport-db-service`
GCP service account we created in step 3.
```

Reply:
nice, i like less text.